package.json

```diff
@@ -28,17 +28,18 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
-    "@sinclair/typebox": "0.32.14",
-    "ajv": "8.12.0",
     "fast-glob": "3.3.2",
+    "lodash": "^4.17.21",
```

Comment:
We could remove the `lodash` and `yup` carets too, to only allow installing `v4.17.21` and `v1.4.0`, respectively.